### ISSUE TITLE
Ratford doesn't start talking to the player automatically.

### ISSUE THREAD

**Issue by memlostexc** [AmProsius/gothic-1-community-patch#389]
**Describe the bug**
At the very beginning of the game Ratford doesn't start talking to the player automatically.

**Expected behavior**
Ratford should start talking to the player automatically.

**Steps to reproduce the issue**
Start the game from the beginning. Go to Ratford.

**Comment by szapp** [comment]
Thanks for the report!
I did a quick test running the first few minutes of the game under different conditions. It seems that this is not related to the G1CP. Gothic (unlike Gothic 2) gracefully let's NPCs finish their current routine before reacting to dialogs. You may notice that sometime NPCs take their time to react after you initiate a dialog. Here, the cause might be that the perceptions are not firing continuously during the small-talk routine. If you wait long enough and dance around him, he will eventually speak to you. How long this takes depends on his routine. I experienced the same regardless of the G1CP.

Bottom line, this is not a G1CP bug. We could, however, address this as a general issue in the G1CP. In due time, I will adjust the title and wording of this issue. -- Thank you for reporting this!

**Comment by AmProsius** [comment]
May be related to #209.
